cmd/crictl/sandbox.go

```diff
@@ -0,0 +1,225 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/urfave/cli"
+	"golang.org/x/net/context"
+	pb "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
+)
+
+var runPodSandboxSpec string
+var podSandboxID string
+var filter string
+
+var runtimeRunPodSanboxCommand = cli.Command{
+	Name:  "runsandbox",
+	Usage: "run a sandbox",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:        "podsandbox-spec",
+			Usage:       "Path of podsandbox spec file",
+			Destination: &runPodSandboxSpec,
+		},
+	},
+	Action: func(context *cli.Context) error {
+		// Set up a connection to the server.
+		conn, err := getRuntimeClientConnection(context)
+		if err != nil {
+			return fmt.Errorf("failed to connect: %v", err)
+		}
+		defer conn.Close()
+		client := pb.NewRuntimeServiceClient(conn)
+
+		err = RunPodSandbox(client, runPodSandboxSpec)
+		if err != nil {
+			return fmt.Errorf("Run podsandbox failed: %v", err)
+		}
+		return nil
+	},
+}
+
+// RunPodSandbox sends a RunPodSandboxRequest to the server, and parses the returned RunPodSandboxResponse.
+func RunPodSandbox(client pb.RuntimeServiceClient, specfile string) error {
+	spec, err := os.Open(specfile)
+	if err != nil {
+		return fmt.Errorf("open spec file failed: %v", err)
+	}
+	defer spec.Close()
+
+	var podSandboxConfig pb.PodSandboxConfig
+	specParser := json.NewDecoder(spec)
+	err = specParser.Decode(&podSandboxConfig)
+	if err != nil {
+		return fmt.Errorf("Parse podSandboxConfig failed: %v", err)
+	}
+
+	r, err := client.RunPodSandbox(context.Background(), &pb.RunPodSandboxRequest{Config: &podSandboxConfig})
+	if err != nil {
+		return err
+	}
+	fmt.Println("RunPodSandboxResponse:")
+	fmt.Println("podsandboxID: ", r.PodSandboxId)
+	return nil
+}
+
+var runtimeStopPodSanboxCommand = cli.Command{
+	Name:  "stoppodsandbox",
+	Usage: "stop a podsandbox",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:        "podsandbox",
+			Usage:       "ID of podsandbox to stop",
+			Destination: &podSandboxID,
+		},
+	},
+	Action: func(context *cli.Context) error {
+		conn, err := getRuntimeClientConnection(context)
+		if err != nil {
+			return fmt.Errorf("failed to connect: %v", err)
+		}
+		defer conn.Close()
+		client := pb.NewRuntimeServiceClient(conn)
+
+		err = StopPodSandbox(client, podSandboxID)
+		if err != nil {
+			return fmt.Errorf("Stop podsandbox failed: %v", err)
+		}
+		return nil
+	},
+}
+
+// StopPodSandbox sends a StopPodSandboxRequest to server, and parses the returned StopPodSandboxRseponse
+func StopPodSandbox(client pb.RuntimeServiceClient, podsandboxID string) error {
+	_, err := client.StopPodSandbox(context.Background(), &pb.StopPodSandboxRequest{PodSandboxId: podsandboxID})
```

Comment:
Maybe we can print something here to tell users that the sandbox has been stopped.